.github/workflows/go.yml

```diff
@@ -39,13 +39,17 @@ jobs:
 
     - name: Install Helm
       uses: engineerd/configurator@v0.0.1
```

Comment:
try upgrade the version, refer to kubevela

Reply:
Done, thanks.